verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -107,6 +107,73 @@ export default class CasesController {
         }
     };
 
+    /**
+     * Validates and upserts the provided cases.
+     *
+     * Executes three operations on the supplied data:
+     *
+     *   1. Geocodes provided cases as required. If there are any issues
+     *      geocoding (which is done serially -- though we should consider
```

Comment:
FYI in case you didn't know we're using the endpoint that supports batch geocoding with mapbox (https://docs.mapbox.com/api/search/#batch-geocoding), we could rely on it to geocode. Their docs do make it clear what are the limits though... I assume the queries must fit in a URL, if so that limits seems pretty low...

Reply:
Yeah, I wasn't sure how to read that:

> Batch geocoding is only available using the **mapbox.places-permanent** endpoint. If you are interested in using **mapbox.places-permanent** to perform batch geocodes, contact Mapbox sales.

Is that a typo, or are they saying that it's available, but you should contact sales first? I wasn't clear.

Reply:
you have to contact sales to enable the endpoint, I think batch comes with it, we are already whitelisted for that endpoint so batch should be available to us.

Reply:
Got it, SG. One thing that's occurred to me is that we might not want to do this for the sake of caching requests (since each geocode in the batch is billed as a request).